lib/usb-device.js

```diff
@@ -16,7 +16,7 @@ class USBDevice {
   }
 
   // opens the device and sets up the endpoints for out object
-  async claimDevice () {
+  claimDevice () {
```

Comment:
It should only be called by `adb-device`'s `initConnection` function, so not public at all. The function doesn't need to `await` on any other calls so I removed the `async`.  What's the reasoning for making everything `async` even if they don't need to be, just for consistency's sake I guess?